### DISCUSSION TITLE
Export gmsh pos file

### DISCUSSION THREAD

**Discussion by Trophime** [feelpp/feelpp#1677] [Ideas]
I would like to export a field (let's say an estimator based on ZZ like what did Cecile in the past) as a gmsh pos file.
I want to try to adapt the mesh using this estimator in gmsh. 
I use this approach instead of using mmg directly because I need to have the boundaries to be also remeshed.

**Comment by prudhomm** [comment]
@Trophime  our idea in the remeshing framework is to do it within the library without having to use external tool/software
